tests/Transport/HttpClientTransportTest.php

```diff
@@ -58,21 +65,22 @@ public function testCreateIndex(): void
     public function testHttpClientIsCalledOnSearch()
     {
         $responses = [
-            new MockResponse(<<<JSON
-{
-  "took" : 1,
-  "timed_out" : false,
-  "hits" : {
-    "total" : {
-      "value" : 0,
-      "relation" : "eq"
-    },
-    "max_score" : null,
-    "hits" : [ ]
-  }
-}
-JSON
-),
+            new MockResponse(
+                <<<'JSON'
+                    {
+                      "took" : 1,
+                      "timed_out" : false,
+                      "hits" : {
+                        "total" : {
+                          "value" : 0,
+                          "relation" : "eq"
+                        },
+                        "max_score" : null,
+                        "hits" : [ ]
+                      }
+                    }
+                    JSON
```

Comment:
IIRC, on Windows it could break the test. Do we need to care about?

Reply:
I don't understand :/ why would it breaks test?